Gems/ROS2/Code/Source/VehicleDynamics/Utilities.h

```diff
@@ -57,4 +57,13 @@ namespace ROS2::VehicleDynamics::Utilities
     //! @returns EntityComponentIdPair that contains id of Hinge component and @param wheelEntityId
     AZ::EntityComponentIdPair GetWheelPhysxHinge(const AZ::EntityId wheelEntityId);
 
+    //! Computes ramped velocity.
+    //! @param targetVelocity last commanded velocity to send to robot (in eg m/s or rad/s)
```

Comment:
nit: Please start comment after variable name with an upper case letter to make it clear this is the start of the comment, so for example the below would become...

```
//! @param targetVelocity Last commanded velocity to send to robot (in eg m/s or rad/s).
```

etc...